packages/@ama-sdk/schematics/package.json

```diff
@@ -54,17 +54,24 @@
     },
     "@openapitools/openapi-generator-cli": {
       "optional": true
+    },
+    "openapi-types": {
+      "optional": true
+    },
+    "type-fest": {
```

Comment:
isn't `type-fest` used only for jest test and so be only in devDep ?

Reply:
Same comment for this addition in other package

Reply:
Nop, it is used to exported Schematics and in CLIs